### ISSUE TITLE
Interface to use bots as a library

### ISSUE THREAD

**Issue by ghost** [certtools/intelmq#1407]
Idea:

```python
>>> from intelmq.bots.experts.ripe.expert import RipeExpertBot
>>> RipeExpertBot.run(ip='83.136.38.146')['source.geolocation.cc']
"AT"
```

**Comment by aaronkaplan** [comment]
added to intelmq 3.0 architecture

**Comment by sebix** [comment]
addressed by PR #2358

**Comment by sebix** [comment]
fixed by #2358
